manim/scene/vector_space_scene.py

```diff
@@ -79,20 +79,90 @@ def add_plane(self, animate: bool = False, **kwargs):
         self.add(plane)
         return plane
 
-    def add_axes(self, animate: bool = False, color: bool = WHITE, **kwargs):
+    def add_axes(
+        self,
+        animate: bool = False,
+        animation: Animation = Create,
+        animation_kwargs: dict[str, Any] | None = {},
+        play_kwargs: dict[str, Any] | None = {},
+        axis_config: dict[str, Any] | None = {},
+        **axis_kwargs: dict[str, Any] | None,
+    ):
         """
         Adds a pair of Axes to the Scene.
 
         Parameters
         ----------
         animate
             Whether or not to animate the addition of the axes through Create.
-        color
-            The color of the axes. Defaults to WHITE.
-        """
-        axes = Axes(color=color, axis_config={"unit_size": 1})
+
+        animation
+            The animation used to add the axes.
+
+        animation_kwargs
+            Any valid keyword arguments of animation.
+
+        play_kwargs
+            Any valid keyword arguments of play.
+
+        axis_config
+            Any valid keyword arguments of Axes.
+
+        axis_kwargs
+            Any valid keyword arguments of Axes.
+
+        Returns
+        -------
+        Axes
+            The Axes object.
+
+        Examples
+        --------
+        .. manim:: AddingAxes
+            class AddingAxes(VectorScene):
+                def construct(self):
+                    axis_config = {"include_tip": True, "unit_size": 2, "include_numbers": True}
+                    animation = FadeIn
+                    animation_kwargs = {"shift": LEFT}
+                    play_kwargs = {"run_time": 1, "rate_func": linear}
+
+                    axes = self.add_axes(
+                        animate=True,
+                        animation=animation,
+                        animation_kwargs=animation_kwargs,
+                        play_kwargs=play_kwargs,
+                        axis_config=axis_config,
+                        x_range=[0, 10, 1],
+                        y_range=[-2, 6, 1],
+                        x_length=5,
+                        y_length=5,
+                    )
+                    self.play(axes.animate.shift(LEFT))
+                    self.wait()
+
+        .. manim:: AddingAxes2
+            class AddingAxes2(VectorScene):
+                def construct(self):
+                    axis_config = {"include_tip": True, "unit_size": 2, "include_numbers": True}
+                    axes = self.add_axes(animate=True, axis_config=axis_config)
+                    self.play(axes.animate.shift(LEFT))
+                    self.wait()
+
+        """
+        axis_kwargs = merge_dicts_recursively(
```

Comment:
This could be marked in docs that there are default parameters for axis settings, and to change it the user have to pass it explicitly